_includes/en/Scripts.md

```diff
@@ -0,0 +1,29 @@
+Daml script is a scripting language to run Daml commands against a ledger. For example:
+
+```
+module Test where
+
+import Daml.Script
+
+test = Script ()
+test = do 
+  alice <- allocateParty "Alice" 
+  bob <- allocateParty "Bob"
+  c <- submit alice $ createCmd NewContract with ...
+  submit bob $ exerciseCmd c Accept with ...
+```
+
+Scripts are compiled like usual Daml code to a `dar` package with the `daml build` command.
+
+| -------------------- | ----------------------------------------------------- |
+| Running a script | `daml script --dar example-0.0.1.dar --script-name ScriptExample:test --ledger-host localhost --ledger-port 6865`
```

Comment:
Could this be `ModuleName:scriptFunction` so it's more clear to people how to refer to their scripts?

Reply:
committed your suggestions.